app/controllers/courses_controller.rb

```diff
@@ -1,16 +1,25 @@
 class CoursesController < InheritedResources::Base
   load_and_authorize_resource
 
+  def create
+    @course = Course.new(course_params)
+    if @course.save
+      redirect_to courses_path, notice: 'Course saved.'
+    else
+      render :new
+    end
+  end
+
```

Comment:
This actually isn't necessary, thanks to [Inherited Resources](https://github.com/josevalim/inherited_resources) and `load_and_authorize_resource`. That said, our controllers have gained much more logic and Inherited Resources no longer covers what we need for many of our actions, not to mention that it is no longer maintained and its use is discouraged. Maybe this is a good time to rip it out completely, and change `load_and_authorize_resource` to just `authorize_resource` in all of our controllers.